driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTestUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.unified;
+
+import org.bson.BsonDocument;
+
+public abstract class UnifiedTestUtils {
```

Comment:
Q: Why abstract? Why a new file - theres already two Helpers classes.

Reply:
Good catch! I opted for an abstract class to allow the two Helper classes to use the common method. Renamed it to `UnifiedHelper` for clarity. If you have any suggestions, please let me know!